File: old_backend/src/main/java/com/jugglehive/backend/model/entity/ttrpg/CharacterInfo.java
package com.jugglehive.backend.model.entity.ttrpg;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Data
@Table(schema = "ttrpg")
public class CharacterInfo {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "lvl")
    private Integer level;

    @Column(name = "current_hp")
    private Integer currentHp;

    @Column(name = "shield")
    private Integer shield;

    @Column(name = "max_hp")
    private Integer maxHp;

    @Column(name = "current_mp")
    private Integer currentMp;

    @Column(name = "current_ki")
    private Integer currentKi;

    @Column(name = "current_fury")
    private Integer currentFury;

    @Column(name = "current_miracles")
    private Integer currentMiracles;

    @Column(name = "current_metamorphs")
    private Integer currentMetamorphs;

}

File: JuggleHiveWebapp.Server/Models/CharacterInfo.cs
using System;
using System.Collections.Generic;

namespace JuggleHiveWebapp.Server.Models;

public partial class CharacterInfo
{
    public long Id { get; set; }

    public int? Lvl { get; set; }

    public int? CurrentHp { get; set; }

    public int? Shield { get; set; }

    public int? MaxHp { get; set; }

    public int? CurrentMp { get; set; }

    public int? CurrentKi { get; set; }

    public int? CurrentFury { get; set; }

    public int? CurrentMiracles { get; set; }

    public int? CurrentMetamorphs { get; set; }

    public virtual ICollection<Chara> Charas { get; set; } = new List<Chara>();
}
